docs/using.rst

```diff
@@ -9,34 +9,46 @@ Getting Certbot
 ===============
 
 To get specific instructions for installing Certbot on your OS, we recommend
-visiting certbot.eff.org_. If you're offline, you can find some general
-instructions `in the README / Introduction <intro.html#installation>`__
+visiting certbot.eff.org_. For general information on how to install Certbot, 
+and the difference between the ``certbot``, ``certbot-auto``, ``letsencrypt``, 
+and ``letsencrypt-auto`` commands, please refer to the 
+`README / Introduction <intro.html#installation>`__
 
 __ installation_
 .. _certbot.eff.org: https://certbot.eff.org
 
 .. _certbot-auto:
 
-The name of the certbot command
--------------------------------
-
-Many platforms now have native packages that give you a ``certbot`` or (for
-older packages) ``letsencrypt`` command you can run. On others, the
-``certbot-auto`` / ``letsencrypt-auto`` installer and wrapper script is a
-stand-in. Throughout the documentation, whenever you see references to
-``certbot`` script/binary, you should substitute in the name of the command
-that certbot.eff.org_ told you to use on your system (``certbot``,
-``letsencrypt``, or ``certbot-auto``).
+Commands
+========
 
+The Certbot client uses a number of different "commands" (also referred
+to, equivalently, as "subcommands") to request specific actions such as
+obtaining, renewing, or revoking certificates. Some of the most important
+and most commonly-used commands will be discussed throughout this
+document; an exhaustive list also appears near the end of the document.
 
 Plugins
 =======
 
 The Certbot client supports a number of different "plugins" that can be
-used to obtain and/or install certificates.  Plugins that can obtain a cert
-are called "authenticators" and can be used with the "certonly" command.
-Plugins that can install a cert are called "installers".  Plugins that do both
-can be used with the "certbot run" command, which is the default.
+used to obtain and/or install certificates.
+
+Plugins that can obtain a cert are called "authenticators" and can be used with
+the "certonly" command. This will carry out the steps needed to validate that you
+control the domain(s) you are requesting a cert for, obtain a cert for the specified
+domain(s), and place it in the ``/etc/letsencrypt`` directory on your
+machine - without editing any of your server's configuration files to serve the 
+obtained certificate. If you specify multiple domains to authenticate, they will
+all be listed in a single certificate. To obtain multiple seperate certificates
+you will need to run Certbot multiple times.
+
+Plugins that can install a cert are called "installers" and can be used with the 
+"install" command.  These plugins can modify your webserver's configuration to
+serve your website over HTTPS using certificates obtained by certbot. 
+
+Plugins that do both can be used with the "certbot run" command, which is the default
+when no command is specified.
```

Comment:
Though you can use the "run" subcommand with a combination of distinct authenticator and installer plugins.